README.md

```diff
@@ -384,22 +322,49 @@ Use the `configlet` tool to generate a README from shared metadata:
 
 If adding a new exercise:
 
-* a [generator](#implementing-a-generator) should be implemented.
-* a minimal, partial, solution should be able to be pushed, in order to create a
-  WIP pull request.
+* a [generator][#implementing-a-generator] should be implemented.
+* a minimal, partial, solution should be able to be pushed, in order to create a WIP pull request.
 
-For an in-depth discussion of how exercism language tracks and exercises work,
-please see the
-[contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data).
+For an in-depth discussion of how exercism language tracks and exercises work, please see the [contributing guide][contributing guide].
 
-If you're just getting started and looking for a helpful way to get involved,
-take a look at
-[regenerating the test suites](#regenerating-a-test-suite),
-[porting an exercise from another language](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md),
-or [creating an automated test generator](#implementing-a-generator).
+If you're just getting started and looking for a helpful way to get involved, take a look at [regenerating the test suites][#regenerating-a-test-suite], [porting an exercise from another language][port exercise], or [creating an automated test generator][#implementing-a-generator].
 
 ## Ruby icon
+
 The Ruby icon is the Vienna.rb logo, and is used with permission. Thanks Floor
 Dress :)
 
+[1]: https://github.com/exercism/ruby/workflows/Configlet%20CI/badge.svg
+[2]: https://github.com/exercism/ruby/actions?query=workflow%3A%22Configlet+CI%22+branch%3Amaster
+[3]: https://github.com/exercism/ruby/workflows/Exercise%20tests/badge.svg
+[4]: https://github.com/exercism/ruby/actions?query=workflow%3A%22Exercise+tests%22+branch%3Amaster
+[5]: https://badges.gitter.im/exercism/ruby.svg
+[6]: https://gitter.im/exercism/ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
```

Comment:
Why use numbers here instead of text?

Reply:
I'mage links I think was my motivation.  (The badge.svg repeats, so this avoids that.)

Reply:
Yeah, it was the "badge area" and so not really all that relevant to the content, like the other links.

Reply:
I'd still prefer text I think:

```suggestion
[configlet-workflow-badge]: https://github.com/exercism/ruby/workflows/Configlet%20CI/badge.svg
[configlet-workflow]: https://github.com/exercism/ruby/actions?query=workflow%3A%22Configlet+CI%22+branch%3Amaster
[tests-workflow-badge]: https://github.com/exercism/ruby/workflows/Exercise%20tests/badge.svg
[tests-workflow]: https://github.com/exercism/ruby/actions?query=workflow%3A%22Exercise+tests%22+branch%3Amaster
[ruby-gitter-badge]: https://badges.gitter.im/exercism/ruby.svg
[ruby-gitter-channel]: https://gitter.im/exercism/ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
```